table/server/master/src/main/java/alluxio/master/table/AlluxioCatalog.java

```diff
@@ -246,6 +269,8 @@ public boolean processJournalEntry(Journal.JournalEntry entry) {
     } else if (entry.hasAddTable()) {
       Database db = mDBs.get(entry.getAddTable().getDbName());
       return db.processJournalEntry(entry);
+    } else if (entry.hasDetachDb()) {
+      apply(entry.getDetachDb());
```

Comment:
this method needs to return `true` if this journal entry has been handled.

Reply:
done